### ISSUE TITLE
bug: error while building Named export Websocket not found

### ISSUE THREAD

**Issue by softwarecurator** [wagmi-dev/viem#1202]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Package Version

1.10.12

### Current Behavior

Errors while building 

```bash
node:internal/event_target:1016
  process.nextTick(() => { throw err; });
                           ^
file:///Users/screw/projects/gamblr/gamblronbase/node_modules/unws/src/node.mjs:1
export { WebSocket } from 'ws'
         ^^^^^^^^^
SyntaxError: Named export 'WebSocket' not found. The requested module 'ws' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'ws';
const { WebSocket } = pkg;
```

### Expected Behavior

build correctly lol

### Steps To Reproduce

npm run build with latest version of viem

**Comment by jxom** [comment]
What environment are you running in? Node, Bun, Deno, Next.js, etc?

**Comment by softwarecurator** [comment]
> What environment are you running in, and what version of that environment? Node, Bun, Deno, Next.js, etc?

node, sveltekit with vercel-adapter downgraded to 1.10.9 and was able to to build fine

**Comment by jxom** [comment]
Thanks. Taking a look.

**Comment by jxom** [comment]
I can't seem to reproduce with my reproduction that mimics your environment (SvelteKit, Node 18, Vercel Adapter). Can you fork my repo and reproduce?

https://github.com/jxom/repro-1202

**Comment by Cali93** [comment]
I'm facing the same issue using `next@13.2.4` with `viem@1.10.12`
I think this bug was introduced with the new `unws` dependency.

**Comment by jxom** [comment]
Can you please provide a minimal reproducible example?

**Comment by jxom** [comment]
Migrated back to `isomorphic-ws` in https://github.com/wagmi-dev/viem/commit/f0708c13b231678fb0abb56b23f8247baa9e3235. Sorry about that.

Should be released in 1.10.13 once build passes.
